crates/compiler/can/src/copy.rs

```diff
@@ -456,6 +465,7 @@ fn deep_copy_expr_help<C: CopyEnv>(env: &mut C, copied: &mut Vec<Variable>, expr
             function_type,
             closure_type,
             return_type,
+            fx_type: effect_type,
```

Comment:
minor: maybe an LSP rename didn't rename `effect_type` to `fx_type`, or maybe intentional